docker/volumeplugin/plugin.go

```diff
@@ -64,12 +66,14 @@ type RancherStorageDriver struct {
 	Basedir         string
 	Scope           string
 	CreateSupported bool
+	InBandAttach    bool
 	Command         string
 	client          *client.RancherClient
 	state           *RancherState
 	mounter         *mount.SafeFormatAndMount
 	FsType          string
 	cli             *dockerClient.Client
+	devices         map[string]string
```

Comment:
Do we have this mapping in metadata or API? I'm more than happy to make things stateless.